src/app/components/Transactions/LogEvent.tsx

```diff
@@ -51,7 +51,7 @@ const DecodedLogEvent: FC<{ layer: Layer; event: RuntimeEvent }> = ({ layer, eve
             <dd>
               <b>{event.evm_log_name}</b>
             </dd>
-            {event.evm_log_params?.length && (
```

Comment:
https://github.com/oasisprotocol/explorer/issues/259
> DecodedLogEvent would print "0" if `event.evm_log_params=[]` because `{event.evm_log_params?.length && (` renders "0".